CMakeLists.txt

```diff
@@ -464,7 +464,6 @@ set(system_test_runner_sources
   "source/tests/system/nidaqmx_session_tests.cpp"
   "source/tests/system/nidcpower_driver_api_tests.cpp"
   "source/tests/system/nidcpower_session_tests.cpp"
-  "source/tests/system/nidigital_driver_api_tests.cpp"
```

Comment:
Can you undo this change now that the tests should be fixed?

Reply:
There are still whitespace differences here.